README.md

```diff
@@ -31,7 +31,29 @@ If you're a developer, you're welcome to submit a pull request.  Please run the
 Use [JSCS](http://jscs.info/) with the AirBnB preset.
 
 ### Jasmine Tests
-Open 'SpecRunner.html' in your web browser to run the test suite.
+Tests are written using [jasmine](https://jasmine.github.io/). They can either be run from the command line using headless chrome or run within a browser window.
+
+First, make sure that node.js and npm are installed.  Then in the project root, run:
+
+```
+npm install
+```
+
+This will install the required packages to run the tests, including a headless version of Chrome that will be controlled via Puppeteer.
+
+To run the headless tests, use:
+
+```
+npm test
+```
+
+To run the tests in the browser, use:
+
+```
+npm test:watch
```

Comment:
does this work yet? I tried locally and the headless command ran fine, but I get the npm help when running this line.

Reply:
D'oh! It should be `npm run test:watch`.  Thanks for catching that!  I'll update the README.